docs/developer-guides/setting-up-token-management-partner-chains.md

```diff
@@ -0,0 +1,402 @@
+# Setting up token management for a partner chain
+
+This guide shows you how to set up and manage a token management contract on the Cardano blockchain, enabling proper observability on partner chains. 
+This guide presents the step-by-step process involving interactions with both the Cardano and the partner chain blockchain.
+
+![Native token management contract](native-token-mgt-contract.png "Native token management contract")
+
+# Prerequisites
+
+- **Cardano Node Setup:** A running Cardano node to interact with the Cardano blockchain
+
+- **Partner Chain Node Setup:** Access to a partner chain node, including Ogmios and Kupo.
+
+- **Development Environment:** Haskell and related tools installed for running scripts.
```

Comment:
This is not a user requirement at all. It is enough to have access to the CLI executable.

Reply:
Haskell (Plutus) is required to write V-function